Seneca-CDOT/telescope
# refactor: fix link to about page in NavBar and remove About page

## Description

This PR fixes the link in the `NavBar` so now it takes you to `/docs/overview`. It also removes the old `about` page.

## Steps to test the PR
1. Run locally
2. Make sure that the `about` link in the `NavBar` (question mark icon) takes you to `/docs/overview`.

## Checklist
